mono/mini/interp/transform.c

```diff
@@ -3765,19 +3762,31 @@ generate (MonoMethod *method, MonoMethodHeader *header, InterpMethod *rtm, unsig
 						}
 						break;
 					case 3:
+						if (MONO_TYPE_IS_VOID (info->sig->ret)) {
+							if (info->sig->params [2]->type == MONO_TYPE_I4)
+								ADD_CODE (td,MINT_ICALL_PPI_V);
+							else
+								ADD_CODE (td,MINT_ICALL_PPP_V);
+						} else {
+							if (info->sig->params [1]->type == MONO_TYPE_I4 &&
+									info->sig->params [2]->type == MONO_TYPE_I4)
+								ADD_CODE (td, MINT_ICALL_PII_P);
+							else
+								g_assert_not_reached ();
```

Comment:
Shouldn't we use `g_assert (info->sig->params [1]->type == MONO_TYPE_I4 && info->sig->params [2]->type == MONO_TYPE_I4)`?

Reply:
@luhenry Yeah, I it looks better that way. I used g_assert_not_reached as a placeholder for not implemented